ui/app/AppMain.qml

```diff
@@ -131,9 +132,16 @@ RowLayout {
               icon.name: "compass"
         }
 
+        StatusIconTabButton {
```

Comment:
I wonder if we shoudn't add a Column { } and Loaders {} for all the StatusIconTabButtons and that way we do not have to deal with the anchors.top property. What do you think, @jrainville?  (this could be done in a separate PR)

Reply:
Yeah I think that could work